common/message/src/main/java/org/thingsboard/server/common/msg/tools/MaxMessageSizeExceededException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright © 2016-2024 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.common.msg.tools;
+
+import lombok.Getter;
+import org.thingsboard.server.common.data.id.DeviceId;
+
+public class MaxMessageSizeExceededException extends RuntimeException {
+
+    @Getter
+    private final DeviceId deviceId;
+
+    public MaxMessageSizeExceededException(DeviceId deviceId) {
+        super(String.format("Max message size exceeded for device [%s]", deviceId));
```

Comment:
I'm not a believer that we need `String.format` here. Just check how complex the implementation is, and you might want to use `string + string + string`